lib/friendly_id/sequentially_slugged.rb

```diff
@@ -76,11 +78,11 @@ def ordering_query
 
     private
 
-    def slug_base_class
+    def slug_base_class_and_column
       if friendly_id_config.uses?(:history)
-        Slug
+        [Slug, :slug]
       else
-        self.class.base_class
+        [self.class.base_class, friendly_id_config.slug_column]
       end
     end
```

Comment:
Why do you think we need those independent ?IMO as it's a private method, this approach saves condition duplication.

Reply:
because otherwise we just split them out when using them: https://github.com/norman/friendly_id/pull/943/files#diff-7f1f46b4c612e77957b8ac31db0290a7R11